extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -247,7 +296,7 @@ export class SystemLayer {
       this.onDesktopStateUpdate_(isDesktop);
     }, true /** callToInitialize */);
 
-    this.storeService_.subscribe(StateProperty.HAS_AUDIO_STATE, hasAudio => {
+    this.storeService_.subscribe(StateProperty.STORY_HAS_AUDIO_STATE, hasAudio => {
       this.onHasAudioStateUpdate_(hasAudio);
```

Comment:
Now that this could be either `onStoryHasAudioUpdate_` or `onPageHasAudioUpdate_`, could you please update this method name?